napari_builtins/io/_read.py

```diff
@@ -133,6 +133,18 @@ def read_zarr_dataset(path: str):
         ]
         assert image, 'No arrays found in zarr group'
         shape = image[0].shape
+    elif (path / 'zarr.json').exists():
```

Comment:
We probably want to add a test for zarr v3? Or is this codecov being silly because only pre is testing v3 arrays?

Reply:
yes. zarr 3 is only available in `--pre` tests